### ISSUE TITLE
Create Ingredient and Recipe DB

### ISSUE THREAD

**Issue by cjohnson19** [CJohnson19/foodle#15]
- [ ] Implement [SQLite](https://sqlite.org/index.html) database for `Ingredient`s, `Recipe`s, and `Pantry`
- [ ] Create logic for linking many `Ingredient`s to a `Recipe`, and many `Ingredient`s to a `Pantry`
- [ ] Create cooking history
- [ ] Migrate current placeholder information to DB

**Comment by cjohnson19** [comment]
Must have #17 before implementation can complete.

**Comment by cjohnson19** [comment]
#17 completed. Waiting on #20 which includes step-by-step instructions on how to cook a recipe.

**Comment by cjohnson19** [comment]
Closing: out of scope.
